main.go

```diff
@@ -70,6 +72,7 @@ func main() {
 		"The namespace where the observability-operator is running.")
 	flag.StringVar(&grafanaURL, "grafana-url", "http://grafana.monitoring.svc.cluster.local",
 		"grafana URL")
+	flag.StringVar(&alloyAppVersion, "alloy-app-version", "0.9.0", "Version of the alloy to use")
```

Comment:
IMO this should be more explicit, something like

```suggestion
	flag.StringVar(&alloyAppVersion, "alloy-rules-app-version", "0.9.0", "Version of the alloy to use for alloy-rules")
```